index.js

```diff
@@ -3,15 +3,20 @@ const async = require('async');
 const Logr = require('logr');
 const path = require('path');
 const fs = require('fs');
+const defaults = require('lodash.defaults');
 const bytesize = require('bytesize');
 
+let DefaultOptions = {
```

Comment:
If I add a lint error like 'var x = 5' right under this line, then my linter correctly identifies all the problems with it.  So there may be something wrong with the capitalization rule in our lint config.